Core/Core/Models/Database.xcdatamodeld/Database.xcdatamodel/contents

```diff
@@ -110,6 +111,7 @@
         <attribute name="id" attributeType="String"/>
         <attribute name="name" attributeType="String"/>
         <relationship name="conversation" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="Conversation" inverseName="participantsRaw" inverseEntity="Conversation"/>
+        <relationship name="conversationMessages" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="ConversationMessage"/>
```

Comment:
I don't see an inverse to this, nor where the association is made. It would also probably need to be a to-many.